{"name": "test_mul_004", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <test_file_005.Test_Credence_005 object at 0x000002C0F8D8B4D0>\n\n    def test_mul_004(self):\n        a = 3\n        b = 7\n        mul = a * b\n        print(\"Mul of a and b is :\" + str(mul))\n        if mul == 10:\n            assert True\n        else:\n>           assert False\nE           assert False\n\ntest_file_005.py:11: AssertionError"}, "attachments": [{"name": "stdout", "source": "c5b2daaa-2713-4fd9-be88-bdc9639c122e-attachment.txt", "type": "text/plain"}], "start": 1690632810324, "stop": 1690632810324, "uuid": "722b9712-b1b2-483d-b598-4f914bf2d761", "historyId": "3373f973f87d181b033c1208fbde30ae", "testCaseId": "3373f973f87d181b033c1208fbde30ae", "fullName": "test_file_005.Test_Credence_005#test_mul_004", "labels": [{"name": "suite", "value": "test_file_005"}, {"name": "subSuite", "value": "Test_Credence_005"}, {"name": "host", "value": "DESKTOP-N95JEM6"}, {"name": "thread", "value": "3988-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_file_005"}]}